### PR TITLE
Password reset

### PR DESCRIPTION
**Pull request by minhpham11**
## Checklist
- [ ] This is bug/feature (delete incorrect one)
- [ ] Title including ticket number and title

## What this PR do?
Explain clearly and briefly what this PR does. Make a meaningful sentence instead of "this PR solve ticket #xxx"

## Context/Background
More information on the issue that you want to provide, related issues and PRs you have to write information to ask orally or in informal place too.

## Screenshots / Demo & Instruction for testing (if applicable)
image

image

- link to demo
- instruction on how to test this

## References
- Blog posts / web pages relevant to this issue (which you get your solution / idea from and it may help others to understand the issue quickly)
- Do we need update our knowledge base / project wiki?

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -62,6 +62,11 @@
     t.datetime "created_at", precision: 6, null: false
     t.datetime "updated_at", precision: 6, null: false
     t.string "remember_digest"
+<<<<<<< HEAD
```
Comment (rikai-tra) [inline db/schema.rb]: conflict

File: app/controllers/password_resets_controller.rb [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+      redirect_to root_url
+    else
+      flash.now[:danger] = "Email address not found"
+      render :new
+    end
+  end
+
+  def update
+    if params[:user][:password].empty?
+      @user.errors.add(:password, "can't be empty")
```
Comment (rikai-hai) [inline app/controllers/password_resets_controller.rb]: đưa vào model

File: app/controllers/password_resets_controller.rb [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+  end
+
+  private
+
+  def user_params
+    params.require(:user).permit :password, :password_confirmation
+  end
+
+  def get_user
+    @user = User.find_by email: params[:email]
```
Comment (rikai-hai) [inline app/controllers/password_resets_controller.rb:40]: tìm ko đc thì sao

**Comment by rikai-tra** [comment]
có thể chỉnh lại cái nút login fb vs google được không?

File: app/controllers/users_controller.rb [hunk 4]
```diff
@@ -7,6 +7,10 @@ def new
   end
 
   def index
+<<<<<<< HEAD
+=======
+    # @users=User.all
+>>>>>>> password_reset
```
Comment (rikai-tra) [inline app/controllers/users_controller.rb]: conflict

File: db/schema.rb [hunk 5]
```diff
@@ -10,7 +10,11 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
+<<<<<<< HEAD
 ActiveRecord::Schema.define(version: 2022_04_10_143702) do
+=======
```
Comment (rikai-tra) [inline db/schema.rb]: chưa hết nè

File: app/controllers/users_controller.rb [hunk 6]
```diff
@@ -7,6 +7,7 @@ def new
   end
 
   def index
+    # @users=User.all
```
Comment (rikai-tando) [inline app/controllers/users_controller.rb]: xóa đi em

**Pull request merged**